modules/nf-core/tcoffee/seqreformat/main.nf

```diff
@@ -0,0 +1,53 @@
+process TCOFFEE_SEQREFORMAT {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::t-coffee=13.46.0.919e8c6b"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/t-coffee:13.46.0.919e8c6b--hfc96bf3_0':
+        'biocontainers/t-coffee:13.46.0.919e8c6b--hfc96bf3_0' }"
+
+    input:
+    tuple val(meta), path(infile)
+    val(seq_reformat_type)
```

Comment:
This shouldn't be in args instead? Also, should it be controlled somehow e.g.:https://github.com/nf-core/modules/blob/b2c97d94552d0760490c8839ac49efc8734dd1ff/modules/nf-core/bamtools/convert/main.nf#L23